GNUmakefile

```diff
@@ -1,4 +1,4 @@
-TEST?=$$(go list ./... |grep -v 'vendor')
+TEST?="./helm"
```

Comment:
This single change allows to get tests output live, as soon as each test is executed, instead of at once at the end of the run when all test have executed.

When there's more than one package under test, the output is buffered to sort the test results alphabetically: https://github.com/golang/go/issues/2731

FYI, this was recently changed on the kubernetes provider: https://github.com/terraform-providers/terraform-provider-kubernetes/commit/be67a1389b77de36875b4e235ce7e859bcd9fb71#diff-b0fa74059a31d3efac99fea6d20ca25aR1

Changing it here avoid the need to override that variable on the command line.

This is supposed to change with Go 1.13 when `-v` is passed to `go test`: https://github.com/golang/go/issues/24929

Reply:
I did the same for K8s.

Reply:
One downside I see is if there are tests in sub-packages, which is the case of the azurerm provider.

Reply:
And there's an even more confusion. This variable is also used in `make test-compile` which does a `go test -c` and that one doesn't work with multiple packages anymore since 1.11